src/python/other-bots/chromium-tests-syncer/run.py

```diff
@@ -235,20 +235,8 @@ def main():
   shell.create_directory(crash_testcases_directory)
   unpack_crash_testcases(crash_testcases_directory)
 
-  # Sync web tests.
-  logs.info('Syncing web tests.')
```

Comment:
It was nice to have a log statement for debugging. Does `clone_git_repository()` do its own logging?

Reply:
Yes it does.

Reply:
Thanks for confirming.